test/SourceKit/CodeComplete/complete_docbrief_package.swift

```diff
@@ -0,0 +1,50 @@
+// BEGIN Module.swift
+package protocol P {
+  /// This is a doc comment of P.foo
+  ///
+  /// Do whatever.
+  func foo()
+}
+
+package struct S: P {
+  public init() {}
+  public func foo() {}
+}
+
+// BEGIN User.swift
+package import DocBriefTest
+
+func test() {
+  S().foo()
+}
+
+// RUN: %empty-directory(%t)
+// RUN: %empty-directory(%t/Modules)
+// RUN: %{python} %utils/split_file.py -o %t %s
```

Comment:
Mind using `%split-file` instead? See the other tests, but it uses `//--- <name>`. Personally I prefer having the `RUN` that needs line next to the actual line itself and then using `%(line+1)`. But I seem to be the only one, so up to you. It would let you move the build to the top of the file though 🤷

Reply:
Sure, updated to use `split-file`, and also updated a couple other variants of this test to do so. Unfortunately it doesn't seem like `%(line+1)` works for anything other than the first split file, I get the error `wrong pos format, line too large`, is that expected? In any case, I've moved the other `RUN` lines to the top of the file, as those won't affect the line position for the split file anyways.

Reply:
You can add `--leading-lines` to handle those cases. It'll add padding newlines to the start of each file so that their lines reflect the original source file.

Reply:
Cool!